official/vision/beta/projects/yolo/modeling/backbones/darknet.py

```diff
@@ -133,90 +140,259 @@ def __call__(self, config, kwargs):
+            False
+        ],
+        [
+            'CSPTiny', 'csp_tiny', 1, False, 64, None, 3, 2, 'same', 'leaky',
+            -1, 1, 2, False
+        ],
+        [
+            'CSPTiny', 'csp_tiny', 1, False, 128, None, 3, 2, 'same', 'leaky',
+            -1, 1, 3, False
+        ],
+        [
+            'CSPTiny', 'csp_tiny', 1, False, 256, None, 3, 2, 'same', 'leaky',
+            -1, 1, 4, True
+        ],
+        [
+            'ConvBN', None, 1, False, 512, None, 3, 1, 'same', 'leaky', -1, 1,
+            5, True
+        ],
     ]
 }
 
 DARKNETTINY = {
-    "list_names": LISTNAMES,
-    "splits": {"backbone_split": 14},
-    "backbone": [
-        ["ConvBN", None, 1, False, 16, None, 3, 1, "same", "leaky", -1, 0, False],
-        ["DarkTiny", "tiny", 1, True, 32, None, 3, 2, "same", "leaky", -1, 1, False],
-        ["DarkTiny", "tiny", 1, True, 64, None, 3, 2, "same", "leaky", -1, 2, False],
-        ["DarkTiny", "tiny", 1, False, 128, None, 3, 2, "same", "leaky", -1, 3, False],
-        ["DarkTiny", "tiny", 1, False, 256, None, 3, 2, "same", "leaky", -1, 4, True],
-        ["DarkTiny", "tiny", 1, False, 512, None, 3, 2, "same", "leaky", -1, 5, False],
-        ["DarkTiny", "tiny", 1, False, 1024, None, 3, 1, "same", "leaky", -1, 5, True],
+    'list_names':
+        LISTNAMES,
+    'splits': {
+        'backbone_split': 14
+    },
+    'backbone': [
+        [
+            'ConvBN', None, 1, False, 16, None, 3, 1, 'same', 'leaky', -1, 1, 0,
+            False
+        ],
+        [
+            'DarkTiny', 'tiny', 1, True, 32, None, 3, 2, 'same', 'leaky', -1, 1,
+            1, False
+        ],
+        [
+            'DarkTiny', 'tiny', 1, True, 64, None, 3, 2, 'same', 'leaky', -1, 1,
+            2, False
+        ],
+        [
+            'DarkTiny', 'tiny', 1, False, 128, None, 3, 2, 'same', 'leaky', -1,
+            1, 3, False
+        ],
+        [
+            'DarkTiny', 'tiny', 1, False, 256, None, 3, 2, 'same', 'leaky', -1,
+            1, 4, True
+        ],
+        [
+            'DarkTiny', 'tiny', 1, False, 512, None, 3, 2, 'same', 'leaky', -1,
+            1, 5, False
+        ],
+        [
+            'DarkTiny', 'tiny', 1, False, 1024, None, 3, 1, 'same', 'leaky', -1,
+            1, 5, True
+        ],
     ]
 }
-# pylint: enable=line-too-long
 
 BACKBONES = {
-    "darknettiny": DARKNETTINY,
-    "darknet53": DARKNET53,
-    "cspdarknet53": CSPDARKNET53,
-    "cspdarknettiny": CSPDARKNETTINY
+    'darknettiny': DARKNETTINY,
+    'darknet53': DARKNET53,
+    'cspdarknet53': CSPDARKNET53,
+    'altered_cspdarknet53': CSPADARKNET53,
+    'cspdarknettiny': CSPDARKNETTINY,
+    'csp-large': LARGECSP53,
 }
 
 
-@tf.keras.utils.register_keras_serializable(package="yolo")
+@tf.keras.utils.register_keras_serializable(package='yolo')
 class Darknet(tf.keras.Model):
-  """Darknet backbone."""
+  """ The DarkNet backbone architecture """
```

Comment:
FWIW:
- Here, the class name is "Darknet" (lower-case n)
- In the pydoc you write "DarkNet"
- In the tests you have a function or class whose name includes "DarkNet"

Reply:
Changed to "Darknet". That is what the original repo seems to stylize it as.